jmaqs-utilities/src/main/java/com/magenic/jmaqs/utilities/helper/exceptions/FunctionException.java

```diff
@@ -8,6 +8,8 @@
  * Custom FunctionException, as you can't throw a generic exception.
  */
 public class FunctionException extends Exception {
+  private static final long serialVersionUID = 1;
```

Comment:
comment what this variable does

Reply:
SerialVeriondUID is acceptable use in an exception

Reply:
Keeping per Jason